config/database.yml

```diff
@@ -3,6 +3,7 @@ default: &default
   adapter: postgresql
   encoding: unicode
   username: <%= ENV.fetch("DB_USERNAME", "postgres") %>
+  password: <%= ENV.fetch("DB_PASSWORD", "100100") %>
```

Comment:
In my home computer without this not working

Reply:
@DenisZackharov it looks like not securable. You store your secret keys in github. You can set `DB_PASSWORD` in `.env` file
```suggestion
  password: <%= ENV.fetch("DB_PASSWORD") %>
```

```
# .env
DB_PASSWORD=100100
```